apps/api/v2/src/modules/oauth/oauth-client.controller.ts

```diff
@@ -47,7 +51,10 @@ export class OAuthClientController {
 
   @Get("/")
   @HttpCode(HttpStatus.OK)
-  async getOAuthClients(@Res({ passthrough: true }) res: Response) {
+  @UseGuards(NextAuthGuard)
```

Comment:
We can move `@UseGuards(NextAuthGuard)` below the
```
@Controller({
  path: "oauth-clients",
  version: "2",
})
@UseGuards(NextAuthGuard)
```

to apply it to all routes.